### ISSUE TITLE
It's not clear how to whitelist tags and attributes

### ISSUE THREAD

**Issue by darmenise** [parndt/refinerycms-wymeditor#27]
I need to use, for instance, an `ol` tag with a `start` attribute, like: `<ol start="3">` but it gets sanitized out.
I can't find any initializer with an example of how to whitelist it.

**Comment by darmenise** [comment]
I tried to whitelist the 'allowfullscreen' attribute in youtube embed code, like:

```
<iframe src="https://www.youtube-nocookie.com/embed/YOUTUBEID" frameborder="0" height="480" width="853" allowfullscreen></iframe>
```
defining the [wymeditor.rb initializer](https://github.com/parndt/refinerycms-wymeditor/blob/master/config/initializers/refinery/wymeditor.rb.erb) with no success: `allowfullscreen` always gets sanitized out.

Help please.

**Comment by bricesanchez** [comment]
This should fix your issue : #29 and #28

**Comment by darmenise** [comment]
Nice, thank you.
